### PR TITLE
Fixing an issue in API metrics dashboard

### PR DESCRIPTION
**Pull request by thefrieddan1**
with JoeSecurity

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CRTX-87482

## Description
Widget ID for joe security was off. Joe widget in API metrics hanged for ever in XSOAR8 and was shown as Cisco umbrella in XSOAR6.

## Screenshots
This is the actual change. not sure why github is displaying the entire block as new. vscode is displaying just the relevant rows:
image

### CODE REVIEW

File: Packs/CommonDashboards/ReleaseNotes/1_3_5.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Dashboards
+
+##### API Execution Metrics
+
+- Fixed an issue with Joe Security widget.
```
Comment (ShirleyDenkberg) [inline Packs/CommonDashboards/ReleaseNotes/1_3_5.md:6]:
```suggestion
Fixed an issue with Joe Security widget.
```
Maybe you should explain what the issue was.

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

File: Packs/CommonDashboards/ReleaseNotes/1_3_5.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Dashboards
+
+##### API Execution Metrics
+
+Fixed an issue with Joe Security widget.
```
Comment (ShirleyDenkberg) [inline Packs/CommonDashboards/ReleaseNotes/1_3_5.md:6]: Do you need to mention what the issue was?

File: Packs/CommonDashboards/ReleaseNotes/1_3_5.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Dashboards
+
+##### API Execution Metrics
+
+Fixed an issue with Joe Security widget wrong ID in API metrics dashboard.
```
Comment (DeanArbel) [inline Packs/CommonDashboards/ReleaseNotes/1_3_5.md:6]:
```suggestion
Fixed an issue where Joe Security widget did not load. 
```

**Review by DeanArbel — APPROVED** [review]
Good job

**Pull request merged**